components/Stats.vue

```diff
@@ -374,30 +357,13 @@ export default {
   },
   computed: {
     getCurrentCountyLatestCases() {
-      let offsetDay = 0
```

Comment:
Cumulative cases/deaths now default to the previous day if there are no new cases so we shouldn't need this check.

Reply:
Looks good!